src/session/session_dhandle.c

```diff
@@ -379,9 +380,15 @@ __session_dhandle_sweep(WT_SESSION_IMPL *session)
     TAILQ_FOREACH_SAFE(dhandle_cache, &session->dhandles, q, dhandle_cache_tmp)
     {
         dhandle = dhandle_cache->dhandle;
+        empty_btree = false;
+        if (dhandle->type == WT_DHANDLE_TYPE_BTREE)
+            WT_WITH_DHANDLE(
+              session, dhandle, empty_btree = (__wt_btree_bytes_evictable(session) == 0));
```

Comment:
The ticket says if it returns 0 OR a small value. What's considered a small value @michaelcahill and do we want to try that?

Reply:
ping @michaelcahill - would you mind taking a look at this question?

Reply:
I'm inclined to just make it zero and update the ticket -- that means all pages apart from the root page have been evicted from cache.  Otherwise I don't know what "small value" to choose...

Reply:
Thanks! I'll go ahead and merge this now.